src/AppInstallerCommonCore/ExperimentalFeature.cpp

```diff
@@ -36,6 +38,8 @@ namespace AppInstaller::Settings
             return ExperimentalFeature{ "Argument Sample", "experimentalArg", "https://aka.ms/winget-settings", Feature::ExperimentalArg };
         case Feature::ExperimentalMSStore:
             return ExperimentalFeature{ "Microsoft Store Support", "experimentalMSStore", "https://aka.ms/winget-settings", Feature::ExperimentalMSStore };
+        case Feature::BrowseOrOpenRemoteUrl:
+            return ExperimentalFeature{ "Open Url", "experimentalBrowseOrOpenRemoteUrl", "https://aka.ms/winget-settings", Feature::BrowseOrOpenRemoteUrl };
```

Comment:
>experimentalBrowseOrOpenRemoteUrl [](start = 53, length = 33)

Can we just start naming these without experimental* at the front? My fault for not catching the last one. So just "browseOrOpenRemoteUrl".

Also this should have changes to /doc/Settings.md to describe it.

Tagging @yao-msft as I forgot to mention this in his PR for experimentalMSStore.